ResearchKit/Localized/en.lproj/ResearchKit.strings

```diff
@@ -276,6 +276,7 @@
 "CARDIO_OUTDOOR_INSTRUCTION_TITLE" = "Please move outdoors";
 "CARDIO_OUTDOOR_INSTRUCTION_TEXT" = "Your phone GPS indicates that you are currently indoors. Please step outdoors to before continuing.";
```

Comment:
Dup of one that Woody opened, but yes. :)  Although your's is much more descriptive. ;)